cluster/v1/types.go

```diff
@@ -144,26 +144,33 @@ const (
 
 // ManagedClusterStatus represents the current status of joined managed cluster.
 type ManagedClusterStatus struct {
-	// Conditions contains the different condition statuses for this managed cluster.
+	// conditions contains the different condition statuses for this managed cluster.
 	Conditions []metav1.Condition `json:"conditions"`
 
-	// Capacity represents the total resource capacity from all nodeStatuses
+	// capacity represents the total resource capacity from all nodeStatuses
 	// on the managed cluster.
 	Capacity ResourceList `json:"capacity,omitempty"`
 
-	// Allocatable represents the total allocatable resources on the managed cluster.
+	// allocatable represents the total allocatable resources on the managed cluster.
 	Allocatable ResourceList `json:"allocatable,omitempty"`
 
-	// Version represents the kubernetes version of the managed cluster.
+	// version represents the kubernetes version of the managed cluster.
 	Version ManagedClusterVersion `json:"version,omitempty"`
 
-	// ClusterClaims represents cluster information that a managed cluster claims,
+	// clusterClaims represents cluster information that a managed cluster claims,
 	// for example a unique cluster identifier (id.k8s.io) and kubernetes version
 	// (kubeversion.open-cluster-management.io). They are written from the managed
 	// cluster. The set of claims is not uniform across a fleet, some claims can be
 	// vendor or version specific and may not be included from all managed clusters.
 	// +optional
 	ClusterClaims []ManagedClusterClaim `json:"clusterClaims,omitempty"`
+
+	// managedNamespaces are a list of namespaces managed by the clustersets the
+	// cluster belongs to.
+	// +optional
+	// +listType=map
+	// +listMapKey=clusterSet
```

Comment:
Since the 'clusterSet' is the key of the map, does that mean for each clusterset only one item is allowed in the list? in another world, only one managed namespace is allowed for a clusterset in the 'managedNamespaces' list?

Reply:
correct, add name is another key